src/config/theme.c

```diff
@@ -452,6 +452,11 @@ _load_preferences(void)
         prefs_set_occupants_size(occupants_size);
     }
 
+    if (g_key_file_has_key(theme, "ui", "occupants.indent", NULL)) {
+        gint occupants_indent = g_key_file_get_integer(theme, "ui", "occupants.indent", NULL);
+        prefs_set_occupants_indent(occupants_indent);
```

Comment:
Doesn't it overwrites the user preference?

Reply:
Yeah it does. But so far it always does overwrite the userprefs with the theme one if you load a theme.